internal/table/console_test.go

```diff
@@ -0,0 +1,63 @@
+package table
+
+import (
+	"bufio"
+	"bytes"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTabularWriter_Write(t *testing.T) {
+	consoleSize = func() (int, int) {
+		return 50, 3
+	}
+	exampleCells := []interface{}{"someValue", 5.0, time.Date(1994, 8, 30, 0, 0, 0, 0, time.UTC)}
+	exampleLine := "|someValue      |5              |1994-08-30 0...|"
+	buffer := bytes.NewBuffer(nil)
+	reader, writer := bufio.NewReader(buffer), buffer
+	w := NewTableWriter(writer)
+	steps := []struct {
+		desc               string
+		cells              []interface{}
+		changeTerminalSize func()
```

Comment:
Is `changeTerminalSize` some kind future investment? I couldn't find where and how it is used.

Reply:
Yes, I know it is not used right know but I wanted to include it just as a reminder to write tests that manipulate terminal size in the future. If you feel uncomfortable with that approach, I can gladly revert it 😊